spec/models/social_networking/comment_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "spec_helper"
+
+module SocialNetworking
+  describe Comment, type: :model do
+    fixtures(:participants, :"social_networking/goals")
+    let(:participant1) { participants(:participant1) }
+    let(:goal) { social_networking_goals(:participant2_goal_alpha) }
+
+    it "should return the number of comments made today" do
+      count = Comment.for_today.count
+
+      Comment.create(
+        participant_id: participant1.id,
```

Comment:
I found that when I referenced the participant directly, the comment didn't seem to be created, and the spec failed. When I got the id directly the specs pass, having created the correct number of comments.